src/lib.rs

```diff
@@ -103,60 +106,65 @@ pub mod pallet {
 			+ GetDispatchInfo
 			+ From<frame_system::Call<Self>>;
 
-		/// The amount of balance that must be deposited per bytes stored.
+		/// The amount of balance that must be deposited per bytes stored
 		#[pallet::constant]
-		type PricePerByte: Get<BalanceOf<Self>>;
+		type DepositPerByte: Get<BalanceOf<Self>>;
 
-		///the maximum users allowed per transaction (adding, removing...)
+		/// The maximum number of accout that can added or removed in a single call
 		#[pallet::constant]
-		type MaxUsersPerTransaction: Get<u32>;
+		type MaxMembersPerTransaction: Get<u32>;
```

Comment:
Maybe we should rename it `MaxAccountPerTransaction`, to be coherent with the doc comment of the type

Reply:
i mean this:

```
		/// The maximum number of account that can added or removed in a single call <-- the comment say "account"
		#[pallet::constant]
		type MaxMembersPerTransaction: Get<u32>; // <-- the type say "members"

```